remote.go

```diff
@@ -595,6 +592,10 @@ func (o *Remote) UpdateFetchHead() bool {
 	return C.git_remote_update_fetchhead(o.ptr) > 0
 }
 
+func (o *Remote) PruneRefs() bool {
+	return C.git_remote_prune_refs(o.ptr) > 0
+}
```

Comment:
This is called `C.git_remote_prune()`, we should also call the Go method `Remote.Prune()`.